tests/sync/test_admin_status.py

```
@@ -0,0 +1,355 @@
+        from fighthealthinsurance.fax_utils import SonicFax
+
+        mock_get.return_value = _members_response()
+        self.assertTrue(SonicFax().check_health())
+        mock_login.assert_called_once()
+
+    @mock.patch.dict(os.environ, _SONIC_ENV)
+    @mock.patch(_LOGIN, side_effect=Exception("login rejected"))
+    def test_check_health_propagates_login_failure(self, mock_login):
+        from fighthealthinsurance.fax_utils import SonicFax
+
+        with self.assertRaises(Exception):
```

Ruff (0.15.17)

[warning] Do not assert blind exception: `Exception`

(B017)